.env

```diff
@@ -0,0 +1,5 @@
+POSTGRES_USER=dbuser
+POSTGRES_DB=task1
+POSTGRES_PASSWORD=dbpasswd
+POSTGRES_HOST=db
+POSTGRES_PORT=5432
```

Comment:
It should be `.env.example`, `.env` is meant to be in the start-ready environment.